api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -408,6 +410,20 @@ public void unvoidVisit_shouldUnvoidEncountersVoidedWithVisit() {
 		assertEquals(1, encountersByVisit.size());
 	}
 	
+	@Test
+	public void getVisitsByPatient_shouldReturnEmptyListGivenNull() {
+		
+		assertThat(visitService.getVisitsByPatient(null), is(empty()));
+	}
+	
+	@Test
+	public void getVisitsByPatient_shouldReturnEmptyListGivenPatientWithNullPatientId() {
+		
+		Patient p = new Patient();
+		p.setPatientId(null);
```

Comment:
newline. remember the AAA (arrange, act, assert) to make it easy for anyone to directly spot what you are testing and what is just setup.